### PR TITLE
docs: native installation

### PR DESCRIPTION
**Pull request by jon-hagg**
### Purpose
Describe the process for installing the framework natively. The output looks like this

Screen Shot 2021-05-13 at 2 48 38 PM

### Time to review
20 mins

### CODE REVIEW

File: source/user/installation_guide.rst [hunk 1]
```diff
@@ -65,6 +65,52 @@ machine. In this case, the only system requirement is to have `docker`_,
...
+--------
+
+The framework can be used natively after some initial setup. This first requires the following:
+
++ install julia 1.5 and create an environment as described in `REISE.jl`_
++ clone `PowerSimData`_ and `REISE.jl`_ (preferably in the same directly)
++ install the solver you wish to use within your julia environment (either Gurobi or GLPK)
+
+Create a virtualenv and activate it using the following. For the activate step, it's important
+to follow exactly (instead of using the "source" command) - this ensures that `sys.executable` will give back the python
```
Comment (rouille) [inline source/user/installation_guide.rst:78]: I think what you want here is ``sys.executable``, i.e., verbatim, and in ReST it is activated with double quote \`\`

File: source/user/installation_guide.rst [hunk 2]
```diff
@@ -65,6 +65,52 @@ machine. In this case, the only system requirement is to have `docker`_,
...
+Create a virtualenv and activate it using the following. For the activate step, it's important
+to follow exactly (instead of using the "source" command) - this ensures that `sys.executable` will give back the python 
+interpreter within this environment.
+
+.. code-block:: console
+
+    python -m venv .env
+    . .env/bin/activate
+
+Install PowerSimData packages into a virtual environment, along with the requirements
```
Comment (rouille) [inline source/user/installation_guide.rst:86]: Install PowerSimData dependencies?

**Comment by kasparm** [comment]
@jon-hagg I'm following these instructions on a Mac using your https://github.com/Breakthrough-Energy/PowerSimData/tree/jon/windows branch.
Currently I'm faced with the following error:
``` 
    ERROR: Command errored out with exit status 1:
     command: /Users/kmueller/EGM/temp/testPSDWindows/.env/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/7s/mqf3m8s52l78wg_fyl2x768w0000gn/T/pip-install-i_dt87b8/psycopg2/setup.py'"'"'; __file__='"'"'/private/var/folders/7s/mqf3m8s52l78wg_fyl2x768w0000gn/T/pip-install-i_dt87b8/psycopg2/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base pip-egg-info
         cwd: /private/var/folders/7s/mqf3m8s52l78wg_fyl2x768w0000gn/T/pip-install-i_dt87b8/psycopg2/
    Complete output (23 lines):
    running egg_info
    creating pip-egg-info/psycopg2.egg-info
    writing pip-egg-info/psycopg2.egg-info/PKG-INFO
    writing dependency_links to pip-egg-info/psycopg2.egg-info/dependency_links.txt
    writing top-level names to pip-egg-info/psycopg2.egg-info/top_level.txt
    writing manifest file 'pip-egg-info/psycopg2.egg-info/SOURCES.txt'

    Error: pg_config executable not found.

    pg_config is required to build psycopg2 from source.  Please add the directory
    containing pg_config to the $PATH or specify the full executable path with the
    option:

        python setup.py build_ext --pg-config /path/to/pg_config build ...

    or with the pg_config option in 'setup.cfg'.

    If you prefer to avoid building psycopg2 from source, please install the PyPI
    'psycopg2-binary' package instead.

    For further information please check the 'doc/src/install.rst' file (also at
    <https://www.psycopg.org/docs/install.html>).

    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

I know how to fix that issue but I can't remember what we decided on how a new user should deal with this?

**Comment by jon-hagg** [comment]
We could move psycopg2 to optional-requirements.txt, as it's not currently used other than tests.

**Review by rouille — APPROVED** [review]
I believe the installation goes smoothly now that the psycopg2 package is moved in the optional PowerSimData requirements in this [PR](https://github.com/Breakthrough-Energy/PowerSimData/pull/480).

File: source/user/installation_guide.rst [hunk 3]
```diff
@@ -65,6 +65,52 @@ machine. In this case, the only system requirement is to have `docker`_,
 All the data and configuration needed to run a simulation are located in the `plug`_
 repository, along with usage examples.
 
+Natively
+--------
+
+The framework can be used natively after some initial setup. This first requires the following:
+
++ install julia 1.5 and create an environment as described in `REISE.jl`_
++ clone `PowerSimData`_ and `REISE.jl`_ (preferably in the same directly)
```
Comment (BainanXia) [inline source/user/installation_guide.rst:74]: in the same directly -> in the same directory?

Comment (rouille) [reply source/user/installation_guide.rst:74]: By the way is it prefearable or mandatory. What happens if you don't do it?

File: source/user/installation_guide.rst [hunk 4]
```diff
@@ -65,6 +65,52 @@ machine. In this case, the only system requirement is to have `docker`_,
 All the data and configuration needed to run a simulation are located in the `plug`_
 repository, along with usage examples.
 
+Natively
+--------
+
+The framework can be used natively after some initial setup. This first requires the following:
+
++ install julia 1.5 and create an environment as described in `REISE.jl`_
```
Comment (ahurli) [inline source/user/installation_guide.rst:73]: Just to be consistent with the rest of the doc, should these bullet points be capitalized?

File: source/user/installation_guide.rst (hunk 3 above)
Comment (jon-hagg) [reply source/user/installation_guide.rst:74]: I'm not actually sure 🙃 not aware of assumptions in the code, but haven't tested in other configurations. Wouldn't be surprised if nesting the repositories or something could cause weirdness, so wanted to encourage following a known setup.

**Review by BainanXia — APPROVED** [review]
I think this is good to go for now.

**Pull request merged**